docs/getting_started.md

```diff
@@ -1,238 +1,204 @@
-unique names (e.g. MyUniqueMLamlcr, MyUniqueML-AML-KV, etc.). The length of
-the BASE_NAME value should not exceed 10 characters and it should contain numbers and letters only.
+You'll need to create a *variable group* in Azure DevOps to store values that are reused across multiple pipelines or pipeline stages. Either store the values directly in [Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=designer#create-a-variable-group) or connect to an Azure Key Vault in your subscription. Check out the [Add & use variable groups](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=yaml#use-a-variable-group) documentation to learn more about how to create a variable group and link it to your pipeline.
 
-The **RESOURCE_GROUP** parameter is used as the name for the resource group that will hold the Azure resources for the solution. If providing an existing AML Workspace, set this value to the corresponding resource group name.
+Navigate to **Library** in the **Pipelines** section as indicated below:
 
-The **AZURE_RM_SVC_CONNECTION** parameter is used by the [Azure DevOps pipeline]((../environment_setup/iac-create-environment-pipeline.yml)) that creates the Azure ML workspace and associated resources through Azure Resource Manager. The pipeline requires an **Azure Resource Manager**
-[service connection](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml#create-a-service-connection).
+![Library Variable Groups](./images/library_variable_groups.png)
 
-![create service connection](./images/create-rm-service-connection.png)
+Create a variable group named **``devopsforai-aml-vg``**. The YAML pipeline definitions in this repository refer to this variable group by name.
 
-Leave the **``Resource Group``** field empty.
+The variable group should contain the following required variables:
 
-**Note:** Creating the ARM service connection scope requires 'Owner' or 'User Access Administrator' permissions on the subscription.
-You must also have sufficient permissions to register an application with
-your Azure AD tenant, or receive the ID and secret of a service principal
-from your Azure AD Administrator. That principal must have 'Contributor'
-permissions on the subscription.
+| Variable Name            | Suggested Value           | Short description                                                                                                            |
+| ------------------------ | ------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| BASE_NAME                | [unique base name]        | Naming prefix                                                                                                                |
```

Comment:
Should this suggest an example name?

Reply:
We had one at some point (I think 'diabetes') but wanted to avoid having diabetes-specific instructions if possible. I'll think about an alternative.

Reply:
I ended up going with the intention of the name (your project name) rather than a specific example. I also added the restrictions as a quick reference.